insert_element

```diff
@@ -0,0 +1,56 @@
+<script>
```

Comment:
Is it possible to provide description about the insert statement and then provide the examples? 
For reference you can take this as an [example](https://github.com/sumn2u/learn-javascript/blob/main/en/basics/comments.md?plain=1).